content/attacks/posts/2023-02-02-Wormhole.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2022-02-03
+custodians: Wormhole
+categories: DeFi
+title: Wormhole Hack: Code Vulnerability Has Led to $325 Million Stolen
+---
+
+## Summary
+
+On February 3, 2022, Wormhole, a DeFi platform that helps move tokens and NFTs across different blockchains (Ethereum, Solana, Binance Smart Chain, etc.), was hacked. The attacker managed to bypass the Wormhole verification with a spoofed sysvar account to mint 120,000 wrapped ETH (wETH) tokens on the Solana network. The invalid 93,750 wETH tokens were used to redeem them back into equivalent ETH tokens on the Ethereum network. The rest of the invalid tokens were swapped for USDC and SOL tokens, as it was shown in the Certic incident analysis report. According to Forbes, the attack caused a 10% drop in the value of the Solana cryptocurrency in the aftermath of the hack. But the worst-case scenario — a bank run and crash of wETH value — did not happen since Jump Trading, Wormhole’s parent company and a major player in the Solana ecosystem, stepped up and supplied Ether to replace what was stolen, after attempts to pay the hacker a $10 million bounty in return for the stolen funds were ignored.
+
+## Attackers
+
+The attackers remain unidentified. The hacker's account: CxegPrfn2ge5dNiQberUrQJkHCcimeR4VXkeawcFBBka and wallet: 0x629e7Da20197a5429d30da36E77d06CdF796b71A.
+
+## Losses
+
+The attacker minted 120,000 wETH — a wrapped Ethereum equivalent on the Solana blockchain — which was equal to $325 million at the time of the theft, making it the largest attack on Solana.
+
+## Timeline:
+
+- **February 2, 2022, 8:15 PM:** Soon after the attack, the Wormhole proposed the hacker a whitehat agreement with a $10 million bounty through the text in a transaction. The hacker didn’t make any contact.
+- **February 3, 2022, 2:42 AM:** Wormhole announced "maintenance" in Twitter while investigating the exploit. 
+- **February 3, 2022, 4:25 AM:** The Wormhole Twitter confirmed the breach and the amount stolen, which was 120k wETH.
+- **February 3, 2022, 6:41 AM:** Wormhole twitted that the vulnerability has been patched.
+
+## Security Failure Causes
+
+The cause of the hack was a bug in the signature verification code, which the intruder tracked by the commit in the Wormhole’s Github repository. This program did not properly check if the signature verification was performed by a whitelisted address, and the team did not have time to deploy the new version at the time of the attack. The attacker substituted the system address with their own program address, bypassing signature verification.
+
+## Additional Details
```

Comment:
[Please avoid non-standard sections](https://github.com/1712n/dn-institute#:~:text=Only%20standard%20sections%20are%20allowed). This part talks about loss recovery and can easily be integrated into the Losses section.

Reply:
Done!

Reply:
Since it's a Crypto Attacks **Wiki**, I suppose it would be great to include the resources used in each article.